playbook/app/pb_kits/playbook/pb_timestamp/docs/_timestamp_updated.jsx

```diff
@@ -5,42 +5,18 @@ const TimestampUpdated = (props) => {
   return (
     <div>
       <Timestamp
-          showUser="true"
+          showUser
           text="Maricris Nonato"
-          timestamp={new Date().getTime()}
-          variant="updated"
-          {...props}
-      />
-
-      <br />
-
-      <Timestamp
-          showUser="false"
-          timestamp={new Date().getTime()}
-          variant="updated"
-          {...props}
-      />
-
-      <br />
-      <br />
-
-      <Timestamp
-          showTimezone="true"
-          showUser="true"
-          text="Maricris Nonato"
-          timestamp={new Date().getTime()}
-          timezone="America/New_York"
+          timestamp={new Date((new Date()).getFullYear() + 4, (new Date().getMonth()), (new Date().getDate()), (new Date().getHours()), (new Date().getMinutes()))}
```

Comment:
This might be a little hard to follow for a doc example. I am having trouble parsing it.

Reply:
Understandable for sure. I had a hard figuring it out in the first place. Not sure how else to go about it though if we want to display the full functionality of the kit. Any ideas? @thestephenmarshall

Reply:
Abstract this inline code to variables would be a great start. Make sure to use meaningful names for the variables so that the doc is sort of self-explanatory.